content/en/docs/concepts/services-networking/endpoint-slices.md

```diff
@@ -147,9 +147,9 @@ In the v1 API, the per endpoint `topology` was effectively removed in favor of
 the dedicated fields `nodeName` and `zone`.
 
 Setting arbitrary topology fields on the `endpoint` field of an `EndpointSlice`
-resource has been deprecated and is not be supported in the v1 API. Instead,
-the v1 API supports setting individual `nodeName` and `zone` fields. These
-fields are automatically translated between API versions. For example, the
+resource has been deprecated and is not going to be supported in the v1 API.
```

Comment:
How about this?
(A tech review that checks whether this suggestion is correct or not is required.)

```suggestion
resource has been deprecated and is not supported in the v1 API.
```

Reply:
That wording LGTM. I'm happy to skip tech review (but still recommend it) as the existing wording is at least as inaccurate.

Reply:
+1 to @jihoon-seo 's [suggestion](https://github.com/kubernetes/website/pull/31598#discussion_r797313656)